app/controllers/invoices_controller.rb

```diff
@@ -7,4 +7,18 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
   end
+
+  def update
+    invoice = Invoice.find(params[:id])
+    if invoice.update(status: params[:status])
```

Comment:
I like the flash messages, maybe I'll go back and add another flash message to the update action in the admin merchant controller for when the status is updated.